tests/integration/kubernetes/k8s-confidential.bats

```diff
@@ -36,7 +36,9 @@ setup() {
 			warn "Failed to get pod IP address."
 		else
 			info "Pod IP address: ${pod_ip}"
-			coco_enabled=$(ssh -i ${SSH_KEY_FILE} -o "StrictHostKeyChecking no" -o "PasswordAuthentication=no" root@${pod_ip} /bin/sh -c "$(get_remote_command_per_hypervisor)") && break
+			base_command="/bin/sh -c"
+			[[ "${KATA_HYPERVISOR}" == "qemu-se" ]] && base_command=""
```

Comment:
Why dropping `"/bin/sh -c"` ?

Reply:
The command for `qemu-se` only worked without `/bin/sh -c` (a default shell was `/bin/ash` as checked). 

This is a solution I ended up with from lots of different combinations of commands to get the command `kubectl exec ...` to return 1 for the entitled guest on the platform. 

So this is the best I can do to achieve my goal. I would be grateful if you suggest any clean and better solution.

Reply:
> So this is the best I can do to achieve my goal. I would be grateful if you suggest any clean and better solution.

From the `SSH(1)` manual page :
```
     If a command is specified, it will be executed on the remote host instead
     of a login shell.  A complete command line may be specified as command,
     or it may have additional arguments.  If supplied, the arguments will be
     appended to the command, separated by spaces, before it is sent to the
     server to be executed.
```

This tends to indicate that `/bin/sh -c` isn't needed at all. I'd rather drop it completely then.

Reply:
tdx is passed, but sev and snp using `dmesg | grep ...` are not picked up by a runner.